contracts/test/mocks/MockRollup.sol

```diff
@@ -62,9 +60,6 @@ contract MockRollup {
         require(validators[msg.sender], "MockRollup.postCommitment: Validator not registered");
         require(commitments[block.timestamp].validator == address(0), "MockRollup.postCommitment: Commitment already posted");
 
-        // verify that the blob header contains the correct quorumBlobParams
```

Comment:
Should we add a TODO to add verification logic back in?

Reply:
I don't think this verification is ever needed. Thoughts @0x0aa0 ?

Reply:
Ya my take is that, at the moment, this contract's purpose is to test happy path blob verification and this check adds unneeded complexity for that. I think at a later point we can revamp this contract to better illustrate some of these security assumptions from the rollups perspective but for now I am OK with removing.